Dockerfile

```diff
@@ -15,7 +15,7 @@ WORKDIR /app/projects/ui
 
 # Install global dependencies and set up for runtime
 RUN apt-get update && apt-get install -y build-essential
-RUN yarn global add vite@4.2.3
+RUN yarn global add vite@4.5.2
```

Comment:
curious about this line.  What are we bringing vite in for?

Reply:
Vite is installed globally here to serve it with `vite preview --port 4000 --host`.

But I think this was leftover from a few versions of the dockerfile ago - changing it to `vite preview --port 4000 --host` and seeing if we can take out this line so we don't need to upgrade the version.

Reply:
Just built it with the new dockerfile changes and the image loaded correctly 🎉